.claude/hooks/deployment-monitor.sh

```
@@ -0,0 +1,140 @@
+    local color="good"
+    local emoji=":white_check_mark:"
+    if [[ "$status" == "FAILED" ]]; then
+        color="danger"
+        emoji=":x:"
+    elif [[ "$status" == "COMPLETE" ]]; then
+        color="good"
+        emoji=":white_check_mark:"
+    fi
+
+    # Create JSON payload
+    local payload=$(cat <<EOF
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)